### PR TITLE
About page fix

### LINKED ISSUE #132: About us -  translation and alignment
The Arabic translation of the about page is out of borders and should be fixed so that it is closer to the middle of the page. Other than that the _Our Team_ text is not translated, translation should be provided

### PR DESCRIPTION
**Pull request by majdajroudi**
This PR adds translation of _Our Team_ text and fixes the alignment of the paragraphs and illustrations in the Arabic version of the about us page 

**Edit:** in the last standup meeting the team decided that we should change the alignment of the text and remove the illustrations because they look weird right now. The social media icons in the cards should be bigger. So those changes are added to this PR
Opera Snapshot_2020-11-23_151936_localhost
#132

### CODE REVIEW

**Review by YamanSammani — APPROVED** [review]
Next time I need to see my picture in the snapshot otherwise I can't approve it 😂😂😂

**Comment by majdajroudi** [comment]
@YamanSammani we'll talk when it's merged *fingers crossed*

**Review by louisrli — COMMENTED** [review]
based on the snapshot though its weird that it goes tarkan louis ammar

you should sort it so that you guys are first

we are last

**Review by louisrli — COMMENTED** [review]
you could add an "order" field in the firebase db then call sort here

**Review by louisrli — APPROVED** [review]
but submit this first

File: src/locales/ar.json [hunk 1]
```diff
@@ -266,7 +267,7 @@
         "role": "مطورة ويب"
       },
       "tarkan": {
-        "name": "طاركان منصوراوغلو",
+        "name": "تاركان منصوراوغلو",
```
Comment (tmansuroglu) [inline src/locales/ar.json:238]: wait do I have an Arabic name?

**Review by tmansuroglu — APPROVED** [review]

File: src/locales/ar.json (hunk 1 above)
Comment (majdajroudi) [reply src/locales/ar.json:238]: I just wrote it in the Arabic alphabet for pronounciation :joy:

**Review by ibrahimaljarad — APPROVED** [review]

**Comment by Ammar-64** [comment]
There’s a conflict here

**Comment by Ammar-64** [comment]
There’s a conflict here

**Comment by majdajroudi** [comment]
i fixed it but it rejecting the push

**Comment by majdajroudi** [comment]
@Ammar-64 PTAL

**Pull request merged**